injected/docs/development-utilities.md

```diff
@@ -4,6 +4,77 @@
 
 To handle the difference in scope injection we expose multiple utilities which behave differently per browser in `src/utils.js` and `ContentFeature` base class. For Firefox the code exposed handles [xrays correctly](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Sharing_objects_with_page_scripts) without needing the features to be authored differently.
 
+## Development Setup
+
+### Repository Access
+
+1. **Clone the repository**: `https://github.com/duckduckgo/content-scope-scripts`
+2. **Request write-access** (for contributors): Submit a request via [Internal Support README](https://app.asana.com/1/137249556945/project/908478224964033/task/1209367367171662?focus=true) asking for `https://github.com/orgs/duckduckgo/teams/core` group access
+
+### Local Development Setup
+
+**Important**: Before cloning the repo on Windows, to avoid major headaches, make sure you clone it with unix-style line endings:
+
+```shell
+git clone --config core.autocrlf=false https://github.com/duckduckgo/content-scope-scripts
+```
+
+The Content Scope Scripts repo includes the build artifacts, which need to be generated as part of your commit.
+
+### Initial Setup
+
+Inside the repo, run:
+
+```shell
+npm ci  # Preferred over 'npm install' for accurate lockfile updates
+```
+
+Ensure you have a version of node that matches what's in the `.nvmrc` file.
+
+Now, to ensure everything's setup, try a full build:
+
+```shell
+npm run build
+```
+
+This will place built files into the top-level `build` folder. If this command ran successfully, you can continue with development.
+
+### Windows Development
+
+Setting up the tools on Windows will be a pain (and really impossible without modifying the scripts, as the build-scripts are not OS-agnostic).
+
+**Recommended**: Use Windows Subsystem for Linux - [WSL Installation Guide](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+Once you have WSL running, make sure you have node and make installed:
+
+```shell
+sudo apt update
+sudo apt install make
+```
```

Comment:
this is only true if we allow new scripts that break the build. I currently build CSS on windows all the time and it works fine with a regular node install. We shouldn't proliferate the use of bash scripts for build (eg: just keep them for releases etc). I don't think the WSL requirement is a good one, it's a barrier to entry that isn't needed if we write cross-platform node scripts instead of bash scripts. especially important given it's a repo of Javascript etc.

For example, right now I can open any terminal in Windows (that has node installed) and I can build CSS (minus a legacy perl thing).